### ISSUE TITLE
Add VGG16 Batchnorm implementation (first part of lesson 5)

### ISSUE THREAD

**Issue by EyalEnav** [recastrodiaz/fastai-course-1-pytorch#2]
Original implementation on courses/deeplearning1/nbs/vgg16bn.py
